pandas/io/sas/sas.pyx

```diff
@@ -367,6 +375,8 @@ cdef class Parser:
     def read(self, int nrows):
         cdef:
             bint done
+
+
```

Comment:
can you remove these extra two lines you've added?

Reply:
Done sir